SalesforceCommerceCloud/pwa-kit
# [commerce-sdk-react] Bug Fix Cookie expires time (@W-16626227@)

# Description



The PR fixes the `commerce-sdk-react` bug https://github.com/SalesforceCommerceCloud/pwa-kit/issues/1991.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Convert the expire number to Date object.

# How to Test-Drive This PR

1. Checkout this branch: `fix-cookie-expires-time` and run `npm ci`
2. Start the template-retail-react-app: `npm start`
3. Open Chrome DevTools Application > Storage > Cookie and verify the Expires time for the `cc-nx-g_RefArchGlobal` cookie is 30 days.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
